sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -1588,8 +1588,102 @@ class HiveDDLSuite
     }
   }
 
+  test("create datasource table with a non-existing location") {
```

Comment:
Let me try to combine the HiveDDLSuite.scala and DDLSuite.scala. Otherwise, the test cases need to be duplicated in every PR.